test/battle/move_effect/bulldoze.c

```diff
@@ -0,0 +1,5 @@
+#include "global.h"
+#include "test/battle.h"
+
+TO_DO_BATTLE_TEST("Bulldoze lowers the speed of every battler hit");
+TO_DO_BATTLE_TEST("Bulldoze's damage is halved when Grassy Terrain is in effect lowers the speed of every battler hit");
```

Comment:
There is no `EFFECT_BULLDOZE` anymore.

Reply:
Forgot to delete this file once I realized this.